go.mod

```diff
@@ -3,11 +3,12 @@ module github.com/open-policy-agent/cert-controller
 go 1.16
 
 require (
-	github.com/onsi/gomega v1.10.2
+	github.com/onsi/gomega v1.15.0
 	github.com/pkg/errors v0.9.1
-	go.uber.org/atomic v1.6.0
-	k8s.io/api v0.20.2
-	k8s.io/apimachinery v0.20.2
-	k8s.io/client-go v0.20.2
-	sigs.k8s.io/controller-runtime v0.8.2
+	go.uber.org/atomic v1.7.0
+	k8s.io/api v0.21.4
+	k8s.io/apiextensions-apiserver v0.21.4
```

Comment:
`go mod tidy` removes this line for me. Looks like you added part of #37?

Reply:
yep, fixed.